### PR TITLE
Add Parsita

### PR DESCRIPTION
**Pull request by drhagen**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by drhagen** [comment]
@conda-forge/help-python, ready for review!

File: recipes/parsita/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  sha256: 0ecfed08b6c95b5403dd7e0f8ef98cfc903b8348a379d16b0841bfa466081cee
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<4.0
```
Comment (beckermr) [inline recipes/parsita/meta.yaml:19]:
```suggestion
    - python >=3.7
```

File: recipes/parsita/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    - poetry-core >=1.0.0
+    - pip
+  run:
+    - python >=3.7,<4.0
```
Comment (beckermr) [inline recipes/parsita/meta.yaml:23]:
```suggestion
    - python >=3.7
```

**Review by beckermr — CHANGES_REQUESTED** [review]
We'll add the `<4` constraint globally to conda-forge if we need it later.

File: recipes/parsita/meta.yaml (hunk 1 above)
Comment (drhagen) [reply recipes/parsita/meta.yaml]: Sounds good to me!

**Review by beckermr — APPROVED** [review]

**Pull request merged**